components/style/properties/data.py

```diff
@@ -134,6 +136,8 @@ class Shorthand(object):
     def __init__(self, name, sub_properties, spec="NO SPEC", experimental=False, internal=False,
```

Comment:
Also, this patch is useless until you change both `spec="NO SPEC"` to `spec=None`, right?